vapi/tags/tag_association.go

```diff
@@ -60,7 +60,19 @@ func (c *Manager) DetachTag(ctx context.Context, tagID string, ref mo.Reference)
 	return c.Do(ctx, url.Request(http.MethodPost, spec), nil)
 }
 
-// AttachTagToMultipleObjects attaches a tag ID to multiple managed objects
+// BatchResponse is the response type used by attach/detach operations which
+// take multiple tagIDs or moRefs as input. On failure Success will be false and
+// Errors contains information about all failed operations
+type BatchResponse struct {
```

Comment:
Since this type isn't returned by the tags.Manager methods, can it be moved to the `vapi/internal` package?

Reply:
Absolutely, I was wondering how to prevent it but needed it for the simulator. So will move accordingly.

Reply:
Made private and used a copy struct in the simulator APIs to avoid cyclic dependency.